windows/src/test/unit-tests/model-ts-parser/assets/custom.model.ts

```diff
@@ -0,0 +1,8 @@
+/*Testing*/
+const source: LexicalModelSource = {
+  format: 'custom-1.0',
+  wordBreaker: 'ascii',
```

Comment:
do you want `custom`?

Reply:
No -- because that means I have to write a function for wordbreaking.